internal/agent/install.go

```diff
@@ -226,6 +226,9 @@ func Install(dir ...string) error {
 }
 
 func RunInstall(c *config.Config) error {
+	utils.SetEnv(c.Env)
+	utils.SetEnv(c.Install.Env)
```

Comment:
do these add on top of the other?

Reply:
Gotcha, thanks! And yeah I guess long term having per action might be a good idea just like you have flags per action